.github/workflows/check-and-build.yml

```diff
@@ -82,5 +82,5 @@ jobs:
       - name: GolangCI Lint
         uses: golangci/golangci-lint-action@v2
         with:
-          version: latest
+          version: v1.39
```

Comment:
Use latest after we upgrade to go1.16, otherwise local build with go1.13 will fail